cms/src/components/content/ContentOverview.vue

```diff
@@ -164,6 +167,54 @@ watch(selectedSortOption, () => {
 onClickOutside(sortOptionsAsRef, () => {
     showSortOptions.value = false;
 });
+
+const tags = db.whereTypeAsRef(DocType.Tag);
+const tagsToDisplay = ref<any[]>([]);
+const tagsSelected = ref([]);
+const tagsContent = ref<ContentDto[]>([]);
+watch([tags, selectedLanguage], async () => {
+    if (!tags.value || tags.value.length === 0) {
+        return;
+    }
+    const tagIds = tags.value!.map((t) => t._id);
+
+    tagsContent.value = await db.whereParent(tagIds, DocType.Tag, selectedLanguage.value);
+
+    tagsContent.value.forEach((tagContent) => {
+        const existingTagIndex = tagsToDisplay.value.findIndex(
+            (tag) => tag.value === tagContent.parentId,
+        );
+
+        if (existingTagIndex === -1) {
+            tagsToDisplay.value.push({
+                label: tagContent.title,
+                value: tagContent.parentId,
+                isChecked: false,
+            });
+        } else {
+            tagsToDisplay.value[existingTagIndex].label = tagContent.title;
+        }
+    });
+});
+watch(tagsSelected.value, () => {
```

Comment:
Is it correct to watch the value of a ref? Or should we rather watch the ref itself?

Reply:
From what I see it is actually a common practice, I used it because I wanted to trigger reactions.

Watching the ref's value (tagsSelected.value) allows you to specifically watch the inner value of the ref. This is useful when you want to trigger a reaction whenever the actual data inside the ref changes.

Watching the ref itself (tagsSelected) allows you to detect changes in the reference itself, such as if the whole ref object is replaced with a new one